BUILDING.md

```diff
@@ -160,6 +160,22 @@ or:
 
 (before the 'make' step)
 
+### Building with CMake
+
+Install Qt and SQLite as instructed in the previous section and clone the repo to a directory (it's `sqlitebrowser` as usual).
+
+And building with these commands:
+
+```bash
+cd sqlitebrowser
+mkdir build       # You can easily make a fresh build again by removing this directory
+cd build
+cmake -DCMAKE_PREFIX_PATH=/usr/local/Cellar/qt/5.11.2_1 ..
+cmake --build .
+```
+
+The `-DCMAKE_PREFIX_PATH=/usr/local/Cellar/qt/5.11.2_1` argument tells CMake where to find Qt libraries. Run `brew info qt` to know where Qt is installed.
```

Comment:
With the above path change, which extra info piece can probably be killed. :smile:

Reply:
Done all. I think we should keep it to briefly explain about this `required` argument since `CMAKE_PREFIX_PATH` has nothing related to Qt :D